cmd/root.go

```diff
@@ -12,7 +13,8 @@ func NewRootCommand() *cobra.Command {
 		Use:   "hermetic",
 		Short: "hermetic - sends and verifies data for digital storage",
 	}
-	rootCommand.PersistentFlags().StringSlice("kafka-endpoints", []string{}, "list of kafka endpoints")
+	rootCommand.PersistentFlags().StringSlice(common_flags.KafkaEndpointsFlagName, []string{}, "list of kafka endpoints")
+	rootCommand.PersistentFlags().String(common_flags.TeamsWebhookNotificationUrlFlagName, "", "url to teams webhook for notifications")
```

Comment:
Utilize Corba for error handling. See also suggested change for `common_flags.go`

```suggestion
	rootCommand.PersistentFlags().StringSliceVar(&common_flags.KafkaEndpoints, "kafka-endpoints", []string{}, "list of kafka endpoints")
	rootCommand.PersistentFlags().StringVar(&common_flags.TeamsWebhookNotificationUrl, "teams-webhook-notification-url", "", "url to teams webhook for notifications")
```